.github/workflows/publish-manifest.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish manifest
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
I think we only want to run this on push to master
```suggestion
```

Reply:
Yeah I just had to test it